app/styles/app.scss

```diff
@@ -36,8 +36,13 @@ body {
 
 .toolbar {
 	height: 60px;
-	border-bottom: 1px solid #e4e2e2;
-	font-size:16px;
+	font-size: 13px;
+  background: #e1563f top center;
+	background-image: url("http://emberjs.com/images/navigation_background.png");
```

Comment:
Seems somewhat odd to hotlink from emberjs.com directly.  I think we should bring this `.png` down, into our own public folder.